labgrid/driver/shelldriver.py

```diff
@@ -180,9 +180,10 @@ def _await_login(self):
             if timeout.expired:
                 raise TIMEOUT(f"Timeout of {self.login_timeout} seconds exceeded during waiting for login")  # pylint: disable=line-too-long
 
+        if self.post_login_settle_time > 0:
+            self.console.settle(self.post_login_settle_time, timeout=timeout.remaining)
```

Comment:
Similar thing here: I think we really want the delay once per activation, but the `self.status` foils that.